draft-kazuho-httpbis-priority.md

```diff
@@ -615,6 +615,15 @@ of delaying the delivery of updates.
 Also, a client MAY use the priority values for making local scheduling choices
 for the requests it initiates.
 
+## Caching of the Priority Response Header
```

Comment:
Is it worth referencing RFC7231 and RFC7234?

Also, I'd prefer to see this in the section about the header, now that that section is much shorter.

Reply:
It references RFC 7234. I've moved the paragraph to the header section, I agree that it would be more natural to explain what the header being end-to-end means, when pointing out that the header is end-to-end.